pkg/controller/worker/helper.go

```diff
@@ -28,6 +28,10 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+const (
+	AwsCsiDriverTopologyKey = "topology.ebs.csi.aws.com/zone"
```

Comment:
why is this exported? If needed, add a doc string

Reply:
The label is being used in `machine_test.go` as well. Sure I will add the docstring

Reply:
If that's the only reason, don't use it in the test. Use the string in the test. This is what we try to do for most test packages. It will enable detecting (potentially undesired) changes of the constant's value.

Reply:
sure, I will do that , thanks.

Reply:
```suggestion
	awsCSIDriverTopologyKey = "topology.ebs.csi.aws.com/zone"
```

Reply:
thanks for that @vpnachev